app/scripts/lib/router.js

```diff
@@ -129,6 +129,7 @@ function (
       'confirm_reset_password(/)': showView(ConfirmResetPasswordView),
       'cookies_disabled(/)': showView(CookiesDisabledView),
       'force_auth(/)': showView(ForceAuthView),
+      'force_auth_complete(/)': showView(ReadyView, { type: 'force_auth' }),
```

Comment:
Not by design. A well caught overlook. 

`account_unlock_complete`, `force_auth_complete`, `reset_password_complete` and `signin_complete` can all check the user's status since the user should have a `sessionToken` (or so we say at this point in time). `signup_complete` may not have a `sessionToken` if the user verifies in a second browser - the 1st browser shows `signup_complete` after `confirm` but has no `sessionToken` while the second browser receives the `sessionToken`.